.gitignore

```diff
@@ -1 +1,3 @@
 tags
+.DS_Store
```

Comment:
Query: What is this file for? Should it be here?

Reply:
these files are annoying artefacts created by MacOS filesystem - https://en.wikipedia.org/wiki/.DS_Store